TheMET/TheMET/ViewControllers/ArtistsCatalogViewController.swift

```diff
@@ -12,63 +12,64 @@ import MetAPI
 
 class ArtistsCatalogViewController: UIViewController {
     
-    private let artistsService = ArtistsService.standart
-    
-    private var artistsServiceSubscriber: AnyCancellable?
-    
-    private let metAPI = MetAPI()
+    private var viewModel: ArtistsCatalogViewModel?
     
     private let loadingArtistsCatalogView = LoadingPlaceholderView.construstView(configuration: .catalogLoading)
     private let failedArtitstCatalogView = FailedPlaceholderView.constructView(configuration: .catalogFailed)
     private let loadedArtistsCatalogView = FeaturedArtistsContentView.constructView()
     
-    private var loadedArtists: [FeaturedArtistsCellData] = []
-    
-    private var loadingArtistNames: [String] = []
+    private var loadedArtistsSubscriber: AnyCancellable?
+    private var contentStatusSubscriber: AnyCancellable?
     
-    var contentStatus:LoadingStatus<[FeaturedArtistsCellData]> = .loading {
-        didSet {
-            self.updateContent()
-        }
-    }
     
     override func viewDidLoad() {
         super.viewDidLoad()
         self.add(artistsCatalogSubview: self.failedArtitstCatalogView)
         self.add(artistsCatalogSubview: self.loadedArtistsCatalogView)
         self.add(artistsCatalogSubview: self.loadingArtistsCatalogView)
+        self.setupViewModel()
+    }
+    
+    private func setupViewModel() {
+        let viewModel = ArtistsCatalogViewModel { [weak self] in
+            return self
+        }
+        self.viewModel = viewModel
+        self.loadedArtistsSubscriber = viewModel.$loadedArtists
```

Comment:
What is this and why do you need it? As I see you control the displayed content including loaded artists with `contentStatus`